pkg/minishift/registration/utils.go

```diff
@@ -30,7 +31,7 @@ type RegistrationParameters struct {
 }
 
 // Register host VM
-func RegisterHostVM(host *host.Host, param *RegistrationParameters) error {
+func RegisterHostVM(host *host.Host, param *RegistrationParameters, proxyInfo string) error {
```

Comment:
-1, Did we not say we will resurrect what we already had. The whole purpose of RegistrationParameters to pass all required parameters.